resources/options.html

```diff
@@ -9,6 +9,7 @@
     <script src="jquery.min.js"></script>
     <script src="materialize.min.js"></script>
     <script src="options.js"></script>
+    <!--TODO: add the do not watch options -->
```

Comment:
Could you explain why this has to be done?
I don't understand why you need the TypeScript file here.